### PR TITLE
[Fix Documentation] update the CONTRIBUTING.md with Issues & Pull Requests sections 

### LINKED ISSUE #313: [Documentation] update the CONTRIBUTING.md with Issues & Pull Requests sections
### Current Behavior

The contributing guide is a great way to introduce new and existing contributors on how to add their contributions to the projects within an organization. Currently, the CONTRIBUTING.md does not state that:

- there is a first-come-first-serve procedure when it comes to assigning open issues.
- allocated time to update the community on your progress

### Desired Situation

Add/update the following sections within the CONTRIBUTING.md of the [BlackPythonDevs](https://github.com/BlackPythonDevs/blackpythondevs.github.io) projects:
- Issues & Pull Requests --> i.e _Creating an Issue, Working on an issue (get it assigned to you)_
- Reviewing Pull Requests

### Checklist for projects
All the CONTRIBUTING.md's need to be updated using above mentioned desired state.

- [x] [BlackPythonDevs](https://github.com/BlackPythonDevs/blackpythondevs.github.io/blob/gh-pages/CONTRIBUTING.md)
- [ ] [Streamlit Booth Slider](https://github.com/BlackPythonDevs/streamlit-booth-slider/blob/CPNTRIBUTING.md)

### PR DESCRIPTION
**Pull request by sandramsc**
This PR fixes #313

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -78,41 +148,155 @@ Help us keep this project open and inclusive. Please read and follow our [Code o
...
+
+  To verify run:
+
+  ```
+  node -v
+  ```
+
+  ```
+  npm -v
+  ```
```
Comment (kjaymiller) [inline CONTRIBUTING.md]: I don't know if these instructions are accurate -

Per the old CONTRIBUTING Doc

> Now that you have the code editor set up, you need to install the dependencies. To do this, you have to open the code editor's terminal and run the command bundle install.

Jekyll is built in Ruby so they would need to install ruby and then run `bundle install`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -78,41 +148,155 @@ Help us keep this project open and inclusive. Please read and follow our [Code o
...
+npm install
+```
+
+**9.** Make the desired changes.
+
+**10.** Run the site locally to preview changes.
+
+```
+npm run build
+```
```
Comment (kjaymiller) [inline CONTRIBUTING.md]: Same issue as previous comment this would  be jekyll and bundle commands

`bundle exec jekyll serve`

**Review by kjaymiller — CHANGES_REQUESTED** [review]
Thanks for getting this in so quickly.

I'm glad we were able to get things linted and corrected. 

In your steps you provide installation instructions for Node and not Ruby. 

Jekyll is a Ruby product so you'll need the installation instructions for ruby and bundle (if you want to go the same level as you did with javascript)

File: CONTRIBUTING.md (hunk 1 above)
Comment (oleksis) [reply CONTRIBUTING.md]: The repository is ready for use with Codespaces or GitPod (Cloud Develpment). We use Devcontainer can read more in this post: [How Can I Contribute to the Black Python Developers Website using a development containers?](https://oleksis.github.io/jupyter/devcontainer/vscode/python/ruby/jekyll/2023/11/05/bpdevs-devcontainer-jekyll-python.html)

Comment (sandramsc) [reply CONTRIBUTING.md]: Hi @kjaymiller, @oleksis, thanks for the feedback, will incorporate the suggestions accordingly.

**Comment by oleksis** [comment]
The [development workflow](https://gist.github.com/oleksis/3dae07876c5de06a13729960270ddf6f#black-python-devs-website) using Devcontainer for the Black Python Devs repository

- [How Can I Contribute to the Black Python Developers Website using a development containers?](https://oleksis.github.io/jupyter/devcontainer/vscode/python/ruby/jekyll/2023/11/05/bpdevs-devcontainer-jekyll-python.html)

**Comment by oleksis** [comment]
When @kjaymiller review it can be merged

**Review by kjaymiller — APPROVED** [review]

**Pull request merged**